content/ru/event-streaming.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Потоковая передача событий
+status: Completed
+category: concept
+tags: ["methodology", "networking", ""]
+---
+
+Потоковая передача событий — это подход, при котором одно приложение отправляет данные о событиях в другие, чтобы постоянно сообщать им о том, что оно делает.
```

Comment:
This would be more close to the original text:
```suggestion
Потоковая передача событий — это подход, при котором программное обеспечение отправляет данные о событиях одного приложения другому, чтобы поддерживать постоянный обмен информацией об их действиях.
```